docs/en/deployment/quick-start.md

```diff
@@ -66,103 +66,99 @@ Quick Start is only for local testing, so generally users do not need to downloa
 2. Execute `mvn clean package -pl apollo-assembly -am -DskipTests=true` in the root directory.
 3. Copy the jar package under apollo-assembly/target and rename it to apollo-all-in-one.jar
 
-# II. Installation steps
-## 2.1 Create the database
-Apollo server side needs a total of two databases: `ApolloPortalDB` and `ApolloConfigDB`, we have prepared the database, table creation and sample data as sql files respectively, just import the database.
-
-> Note: If you have already created Apollo database locally, please take care to backup the data. The sql file we prepared will clear the Apollo related tables.
-
-### 2.1.1 Creating ApolloPortalDB
-Just import [sql/apolloportaldb.sql](https://github.com/apolloconfig/apollo-quick-start/blob/master/sql/apolloportaldb.sql) through various MySQL clients.
+# II. Initialization and Startup
+#### Precautions
+1. The Apollo server process needs to use ports 8070, 8080, 8090 respectively, please ensure these three ports are not currently in use.
+2. The `github` in the SPRING_PROFILES_ACTIVE environment variable in the script is a required profile, `auth` is a profile that provides simple authentication for the portal, it can be removed if authentication is not required or other authentication methods are used.
+## 2.1 Use H2 in-memory database, automatic initialization
+No configuration is required, just use the following command to start
+> Note: When using the in-memory database, any operation will be lost after the Apollo process restarts
+```bash
+export SPRING_PROFILES_ACTIVE="github,auth"
```

Comment:
Is it feasible to enhance the demo.sh script to enable users to initiate the quick-start demonstration efficiently through some configuration settings?